pkg/utils/xml2text.go

```diff
@@ -12,24 +12,24 @@ import (
 	"github.com/pkg/errors"
 )
 
-func XmlNodeAsMarkdownPreRender(node *xmlquery.Node) string {
-	return xmlToMarkdown(node.OutputXML(false), true)
+func XmlNodeAsMarkdownPreRender(node *xmlquery.Node, needsSpace bool) string {
```

Comment:
Does this signature need to change? Or will this function always be called with `needsSpace=False`? Based on the changes in profileparser.go, it looks like this will always be called with `true`.

Reply:
thanks for the review, yes, it will be used here: https://github.com/openshift/compliance-operator/blob/fd0ab5b03eeb70ab45101b8e65518154e11730ef/pkg/utils/parse_arf_result.go#L90

Reply:
We will call false in the warnings section since we don't want extra space in API path

Reply:
Aha - got it, I didn't look at the other review.

Thanks!